### PR TITLE
I-DOCSIS 7.0.3

### PR DESCRIPTION
**Pull request by MariekeGO**

### CODE REVIEW

File: release-notes/EPM/I-DOCSIS/EPM_7.0.3.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+> We are still working on this release. Release notes may still be added, modified, or moved to a later release. Check back soon for updates!
+
+## New features
+
+#### EPM_I_DOCSIS_AddNewCcapCmPair improvements [ID_39450]
+
+Several improvements have been implemented in the EPM_I_DOCSIS_AddNewCcapCmPair script, which is used to create a new CCAP.
+
+- Every page now includes a *Cancel* button, which you can use the terminate the script.
+- Validation on fields now prevents the user from proceeding if fields still need to be filled in.
```
Comment (miguel-slc) [inline release-notes/EPM/I-DOCSIS/EPM_7.0.3.md:17]: Hi,
Maybe we could rephrase this sentence? If I understand correctly the script now validates empty fields. If this is the case we can just say *The script now validates empty fields before to proceed with the provisioning*

Comment (MariekeGO) [reply release-notes/EPM/I-DOCSIS/EPM_7.0.3.md:17]: How about this?
```suggestion
- If fields that must be filled in are still empty, the field validation will now prevent the user from proceeding.
```

Comment (miguel-slc) [reply release-notes/EPM/I-DOCSIS/EPM_7.0.3.md:17]: It looks ok. Thanks @MariekeGO!

File: release-notes/EPM/I-DOCSIS/EPM_7.0.3.md [hunk 2]
```diff
@@ -16,3 +16,19 @@ Several improvements have been implemented in the EPM_I_DOCSIS_AddNewCcapCmPair
...
+### Enhancements
+
+#### Boundaries updated for US/DS thresholds [ID_39507]
+
+For the following parameters in the upstream and downstream threshold tables, the threshold boundaries were updated:
+
+- For the upstream QAM channel for the collector, the Minimum Tx Power Level is now 25.0 dBmV.
+- For the upstream QAM channel for the CCAP, the Maximum Timing Offset Level is now 10000 µs.
+- For the downstream QAM channel for the collector, the Maximum Rx Power Level is now 20 dBmV.
+
```
Comment (miguel-slc) [inline release-notes/EPM/I-DOCSIS/EPM_7.0.3.md:31]: Hi @sergiosuarezsky, can we change *collector* for *CM Collector*?

Comment (miguel-slc) [reply release-notes/EPM/I-DOCSIS/EPM_7.0.3.md:39]: Updates were done on this review

**Pull request merged**